live-site/components/event-schedule-section/event-item/EventItem.styles.ts

```diff
@@ -156,20 +151,51 @@ const StyledPinpoint = styled.img`
   } ;
 `;
 
-const StyledArrow = styled.img`
+const StyledArrow = styled.img<StyledArrowProps>`
   padding-top: 3em;
   padding-right: 3em;
   float: right;
-
+  transform: scaleY(${(StyledArrowProps): string => {
```

Comment:
I think for parameters the convention is to use camel case so 
styledArrowProps